index.html

```diff
@@ -0,0 +1,606 @@
+<!DOCTYPE html>
+<html lang="EN">
+
+<head>
+	<title>TWN</title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+	<link rel="stylesheet" type="text/css" href="resources/css/main.css">
+</head>
+
+<body>
+	<header>
+		<div class="nav-container">
+			<div class="logo"></div>
```

Comment:
This logo is supposed to appear for smaller screens, right?
It seems not to be showing.

-  [ ] TWN logo appears at the leftmost part.


| Current | Source |
| :-----: | :----- |
| ![tnw-current-small](https://user-images.githubusercontent.com/36057474/66247266-aed2f800-e712-11e9-941c-ab57b8ecebf8.png) | ![tnw-small-screen-source](https://user-images.githubusercontent.com/36057474/66247494-27d34f00-e715-11e9-93ea-0e518c8cbab9.png) |